src/wrappers/_MarkdownWrapper.jsx

```diff
@@ -1,65 +1,47 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import TerraMarkdown from 'terra-markdown';
 import classNames from 'classnames/bind';
+
+import CodesplitWrapper from './_CodesplitWrapper';
```

Comment:
Should this be CodeSplitWrapper?

Reply:
It could be. I didn't spell the class that way previously and I don't feel strongly enough about it to change it.